diff --git a/PurePKG/Extensions/ViewModifier.swift b/PurePKG/Extensions/ViewModifier.swift
index 6314110..c8df44f 100644
--- a/PurePKG/Extensions/ViewModifier.swift
+++ b/PurePKG/Extensions/ViewModifier.swift
@@ -139,7 +139,7 @@ extension View {
                 .edgesIgnoringSafeArea(.bottom)
                 .frame(width: appData.size.width)
         )
-        #else
+        #elseif os(iOS)
         self.background(
             VStack {
                 Image("BG")
@@ -151,6 +151,8 @@ extension View {
                 .edgesIgnoringSafeArea(.bottom)
                 .frame(width: UIScreen.main.bounds.width)
         )
+        #elseif os(tvOS)
+        self.background(Color(hex:"#2d003d"))
         #endif
     }

diff --git a/PurePKG/Main.swift b/PurePKG/Main.swift
index 8b40843..5f43077 100644
--- a/PurePKG/Main.swift
+++ b/PurePKG/Main.swift
@@ -351,7 +351,10 @@ struct MainView: View {
 #else
             .frame(width: UIScreen.main.bounds.width)
 #endif
+            #if !os(tvOS)
+            // A8 is TRASH!
             .background(VisualEffectView(effect: UIBlurEffect(style: .dark)).edgesIgnoringSafeArea(.all))
+            #endif
             .transition(.move(edge: .bottom))
             .animation(.spring(), value: appData.queued.all.isEmpty).animation(.spring(), value: queueOpen).animation(.spring(), value: editing).noTabBarBG()
         }
